homeassistant/components/isy994/services.py

```diff
@@ -447,3 +469,43 @@ def async_setup_light_services(hass: HomeAssistant) -> None:
     platform.async_register_entity_service(
         SERVICE_SET_RAMP_RATE, SERVICE_SET_RAMP_RATE_SCHEMA, "async_set_ramp_rate"
     )
+
+
+@callback
+def async_log_deprecated_service_call(
+    hass: HomeAssistant,
+    call: ServiceCall,
+    alternate_service: str,
+    alternate_target: str | None,
+    breaks_in_ha_version: str,
+) -> None:
+    """Log a warning about a deprecated service call."""
+    deprecated_service = f"{call.domain}.{call.service}"
+    alternate_target = alternate_target or "this device"
```

Comment:
Should never get to "this device" but MyPy was failing with `async_get_entity_id` returning an Optional[str].

Reply:
You could also do `assert alternate_target is not None`

Reply:
Thought about it but I figured this way of for some reason it wasn't found at least the warning and issue still go through with a coherent message.